src/server/server_family.cc

```diff
@@ -624,7 +624,7 @@ void ServerFamily::Shutdown() {
 
   if (save_on_shutdown_ && !absl::GetFlag(FLAGS_dbfilename).empty()) {
     shard_set->pool()->GetNextProactor()->Await([this] {
-      GenericError ec = DoSave();
+      GenericError ec = DoSave(true);
```

Comment:
What will happen if we have a running save operation and than a shutdown command is executed?
Is the save process is cancelled and than we continue to shutdown and save again?
will we have 2 save operations running at the same time?

Reply:
I will fix it